4-create_charts.R

```diff
@@ -577,25 +585,29 @@ for (i in 1:nrow(locations)) {
   acc_pet_stdev <- aWhereCharts::generateaWhereStdDevChart(data = weather_df
                                                 ,variable = "accumulatedPet"
                                            ,title = paste(acc_pet_stdev_title
-                                                          ,lat_lon))
+                                                          ,lat_lon
+                                                          ,date_start, "to"
+                                                          ,date_end))
   
   # Daily P/PET -----------------------------------------------------------
   # P/PET = Precipitation-over-PET ratio 
   # (NOTE!!! --> P/PET is rarely is interpretable on a daily chart) 
-  ppet_title <- paste0(place_name,": PPET")
+  ppet_title <- paste0(place_name,": P PET")
```

Comment:
I think these "P PET" strings should be "P/PET"

Reply:
Yes - it definitely should be P divided by PET. The chart titles are currently used to generate the filenames, and we ran into an issue with having "/" in filenames. This is a good fix to add to the future to-do list.

Reply:
Ah gotcha. In that case I'm fine to sign off on this, and we can revisit the filenames in the future.